### PR TITLE
aii-ks: support mailx and SMTP via mailx

### PR DESCRIPTION
**Pull request by stdweird**
Requires #268

### CODE REVIEW

**Comment by JensTimmerman** [comment]
jenkins test this please

**Comment by JensTimmerman** [comment]
jenkins test this please

**Comment by jrha** [comment]
@stdweird any reason not to have this in 17.7?

**Comment by stdweird** [comment]
@jrha no, not really. feel free to review and merge 😄

File: aii-ks/src/main/pan/quattor/aii/ks/schema.pan [hunk 1]
```diff
@@ -60,6 +60,14 @@ type structure_ks_logging = {
 };
 
 @documentation{
+    Configure email settings
+}
+type structure_ks_mail = {
+    @{Send email on succesful install}
+    "success" : boolean = false
+} = dict();
```
Comment (jrha) [inline aii-ks/src/main/pan/quattor/aii/ks/schema.pan]: Is this `dict()` necessary?

**Comment by jrha** [comment]
LGTM

**Pull request merged**